nastafile.asd

```diff
@@ -18,6 +18,7 @@
                "group-by"
                "rutils"
                "log4cl-extras"
-               "trivial-garbage")
+               "trivial-garbage"
+               "defmain")
```

Comment:
Let's add a Deploy here to build a binary using `(asdf:make :nastafile)`.

Here is how to do this: https://github.com/Shinmera/deploy#how-to